lib/mail/fields/content_type_field.rb

```diff
@@ -125,6 +125,10 @@ def sanitize(val)
 
       if val =~ /(boundary=(\S*))/i
         val = "#{$`.downcase}boundary=#{$2}#{$'.downcase}"
+      elsif val =~ /(name=(\S*))/i
```

Comment:
we are repeating this pattern 3 times now, does it make sense to just match on `([a-z]+)` and make the original clause more generic?

Reply:
I went for the minimum change to avoid breaking anything and I am under the belief that a smaller change might be merged faster. I do not honestly understand why the `.downcase` is required at all and if it were up to me, I'd remove the whole `if` block. I hope a maintainer who knows the gem better will clarify things here.

I am thinking that there is some lookup at some point (maybe for `charset`?) which is case-sensitive and that is why this has been added in the first place, but I believe that is the exception rather than the norm.

Reply:
I did not go with the generic suggestion because I am not sure if that will not break something else, but I merged the three regular expressions, so the original clause is now more generic.